### DISCUSSION TITLE
Lazyload images without caorusel

### DISCUSSION THREAD

**Discussion by CROGOT** [fancyapps/ui#224] [Ideas]
Add the ability to use the LazyLoad without a carousel

**Comment by fancyapps** [comment]
Hi,

Sorry, but where would you like to have that ability?

**Comment by CROGOT** [comment]
I would like to use on regular pictures that are not in the carousel. Therefore, I have to use the library https://github.com/aFarkas/lazysizes
But it would be easier to use one library (yours), especially since the implementation is already there (if it works correctly outside the carousel)

**Comment by fancyapps** [comment]
If you do not need to support outdated browsers, then you can implement image lazy loading in just a few lines using Intersection Observer API, for example - https://medium.com/fasal-engineering/image-lazy-loading-using-browsers-intersection-observer-api-a-step-by-step-guide-with-examples-b1a867614e8 (or use small library like https://github.com/mciastek/sal).

Image lazy loading inside carousel is very basic, because carousel checks what slides are visible (for positioning), then it simply replaces attributes like `data-lazy-src` to `src` for elements that are inside these slides.
